File: Rashmi/git.py
import urllib
def readData(url):
    f=urllib.urlopen(url);
    s=f.read()
    s=[x.strip() for x in s.split(',')]
    for k in s:
        #print k
        t=[x for x in k.split(':')]
       # print t
        for l in t:
                print l
                if(l.startswith("https")):
                    print "yo"
   

readData("https://api.github.com/users/rashrag");

File: Rashmi/git.php
<?php

	function getData($file)
	{
		$j="{";
		$f=file_get_contents($file);
		$f1=explode("\n",$f);
		foreach($k as $f1)
		{
			$f2=explode(":",$f1);
			if((substr($f2[0],0,5) == 'https'))
			{
				$fret=getData($f2[1])
				$j=$j.$f2[0].":".$fret;
			}
			else
			{
				$j=$j.$f2[0].":".$f2[1];
			}
			$j=$j."}";
		}
		return $j;
	}
$m=getData("https://api.github.com/users/rashrag");	
echo $m;

?>